dev/NavigationView/NavigationView.xaml

```diff
@@ -181,10 +185,10 @@
                             </VisualStateGroup>
 
                             <VisualStateGroup x:Name="PaneSeparatorStates">
-                                <VisualState x:Name="SeparatorCollapsed"/>
-                                <VisualState x:Name="SeparatorVisible">
+                                <VisualState x:Name="SeparatorVisible"/>
+                                <VisualState x:Name="SeparatorCollapsed">
                                     <VisualState.Setters>
-                                        <Setter Target="VisualItemsSeparator.Visibility" Value="Visible"/>
+                                        <Setter Target="VisualItemsSeparator.Visibility" Value="Collapsed"/>
```

Comment:
Out of curiosity, why is this switched now?

Reply:
oh oops. I was previously getting the separator ActualHeight from codebehind, and for it to evaluate to a non-zero value, it needed to be visible by default. The separator is no longer used for pane layout calculations, so I should revert this.